internal/platform/state/sync/ownership/sync.go

```diff
@@ -24,12 +27,29 @@ func NewService(tx storage.Tx) *service {
 	}
 }
 
+func (s *service) SetOwnershipBlock(blockNumber uint64, block model.Block) error {
+	formatedOwnerhipBlockNumber := formatBlockNumber(blockNumber, blockNumberDigits)
```

Comment:
typo, put `formatedOwnershipBlockNumber` instad of `formatedOwnerhipBlockNumber`